include/DetectorModule.hh

```diff
@@ -224,6 +230,10 @@ public:
 
   const XYZVector& center() const { return decorated().center(); }
   const XYZVector& normal() const { return decorated().normal(); }
+  const XYZVector& cornerone() const { return decorated().cornerone(); }
+  const XYZVector& cornertwo() const { return decorated().cornertwo(); }
+  const XYZVector& cornerthree() const { return decorated().cornerthree(); }
+  const XYZVector& cornerfour() const { return decorated().cornerfour(); }
```

Comment:
Is this a function to extract the module's corners? I thought it already existed.

Reply:
Additionally: please use camel case notation.

Reply:
> Is this a function to extract the module's corners? I thought it already existed.

Yes, it is. I didn't find an existing function, but I'll try to improve it because it's not a very nice solution in any case.